### PR TITLE
Fix lint warnings and specify compiler version

### PR DESCRIPTION
**Pull request by DemiMarie**
Truffle needs a specific compiler version to be specified.  Also, some
lint warnings could be fixed.

### CODE REVIEW

File: contracts/EdgewareERC20.sol [hunk 1]
```diff
@@ -5,20 +5,21 @@ import "openzeppelin-solidity/contracts/token/ERC20/ERC20Mintable.sol";
...
     address public controller;
 
-    constructor() {
+    constructor() public {
         controller = msg.sender;
     }
 
     /**
     * @dev Burns a specific amount of tokens.
+     * @param account The address to burn tokens from.
```
Comment (drewstone) [inline contracts/EdgewareERC20.sol]: Spacing is off by 1.

File: contracts/ValidatorSet.sol [hunk 2]
```diff
@@ -32,27 +32,28 @@ contract ValidatorSet {
...
+        public view returns (bool) {
         uint64 signedPower = 0;
         for (uint i = 0; i < signers.length; i++) {
         if (i > 0) {
-            require(signers[i] > signers[i-1]);
+            require(signers[i] > signers[i-1], "Signers array not sorted");
         }
         address recAddr = ecrecover(hash, v[i], r[i], s[i]);
-        require(recAddr == addresses[signers[i]]);
+            require(recAddr == addresses[signers[i]], "Invalid recipient address");
```
Comment (drewstone) [inline contracts/ValidatorSet.sol]: Spacing is off.

**Review by drewstone — CHANGES_REQUESTED** [review]
Have you run all tests and ensured they pass?

**Comment by DemiMarie** [comment]
@drewstone I tried to run the tests, but failed.  What is the best way to run them?

File: contracts/ValidatorSet.sol (hunk 2 above)
Comment (DemiMarie) [reply contracts/ValidatorSet.sol]: Fixed.

**Comment by DemiMarie** [comment]
@drewstone Running `truffle test` fails on master and on this branch, with obscure errors in both cases.

**Comment by DemiMarie** [comment]
That said, I am not sure what version of Truffle to use.

**Comment by DemiMarie** [comment]
@drewstone The tests now pass on my system (Truffle 5.0.0).

**Review by drewstone — APPROVED** [review]

**Comment by drewstone** [comment]
Look good! Thanks @DemiMarie !

**Pull request merged**